app/views/apcsp/PageMarketing.vue

```diff
@@ -297,6 +297,10 @@ export default Vue.extend({
       setPageTitle: 'teacherDashboard/setPageTitle'
     }),
 
+    checkUsingRoute () {
+      return this.$route
```

Comment:
can you do `this.$route.path === '/teachers/apcsp'` instead as a check?

Reply:
this throws an error because this.$route is undefined in /apcsp
![image](https://github.com/codecombat/codecombat/assets/164280260/e79ddbff-f906-45bb-8152-870dc2d242dc)

Reply:
you can use `this.$route?.path === '/teachers/apcsp'` or you can pass a parameter from router for this and check that parameter